dsx-fw/DSX_Library/ADC/Inc/adc.h

```diff
@@ -13,7 +13,7 @@
 #define NUM_ADC_CHANNELS 4U
 
 // Import adc handle from main.c for use in adc.c
-extern ADC_HandleTypeDef hadc2;
```

Comment:
I think this variable should be define in the header file rather than the source file